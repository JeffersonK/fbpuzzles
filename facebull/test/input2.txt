M1 C1 C2 1000
M2 C2 C1 1000
M3 C3 C2 1000
M4 C2 C3 1000
M5 C3 C1 1000
