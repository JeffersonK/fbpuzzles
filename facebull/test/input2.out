3000
1 4 5
